CONTRIBUTING.rst

```diff
@@ -258,6 +258,13 @@ Please also follow the recommendations from the
 
 Python
 ~~~~~~
+We use `Black <https://black.readthedocs.io/en/stable/>`_ to ensure a consistent code format.
+You can auto-format your code by running:
+
+.. code-block:: bash
+
+    black --check --line-length=100 --exclude=mlflow/protos .
+
```

Comment:
Added how to apply auto-formatting

Reply:
Looks like this checks the formatting, but doesn't actually apply it - I guess we need to run `black --line-length=100 --exclude=mlflow/protos` to apply formatting?

Reply:
Yes, I forgot to remove `--check`.